packages/terra-form-textarea/README.md

```diff
@@ -0,0 +1,25 @@
+# Terra Form Textarea
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-form-textarea.svg)](https://www.npmjs.org/package/terra-form-textarea)
+[![Build Status](https://travis-ci.org/cerner/terra-core.svg?branch=master)](https://travis-ci.org/cerner/terra-core)
+
+Textarea component for building forms.
```

Comment:
Can we update the descriptions to match in both ReadMes & the package.json?

Reply:
Sure can! https://github.com/cerner/terra-core/pull/982/commits/9e911990d3df96486e60680870d9598b0f6d6579